language/de.json

```diff
@@ -0,0 +1,372 @@
+{
+	"Preview": "Vorschau",
+	"Gallery": "Galerie",
+	"Generate": "Generieren",
+	"Skip": "Überspringen",
+	"Stop": "Stop",
+	"Input Image": "Eingabebild",
+	"Advanced": "Erweitert",
+	"Upscale or Variation": "Vergrößern oder Abwandeln",
+	"Image Prompt": "Bild Prompt",
+	"Inpaint or Outpaint (beta)": "Inpainting oder Outpainting (beta)",
+	"Drag above image to here": "Ziehen Sie das obere Bild hierher",
+	"Upscale or Variation:": "Vergrößern oder Abwandeln:",
+	"Disabled": "Deaktiviert",
+	"Vary (Subtle)": "Variieren (Subtil)",
+	"Vary (Strong)": "Variieren (Stark)",
+	"Upscale (1.5x)": "Vergrößern (1.5x)",
+	"Upscale (2x)": "Vergrößern (2x)",
+	"Upscale (Fast 2x)": "Vergrößern (Schnell 2x)",
+	"\ud83d\udcd4 Document": "\uD83D\uDCD4 Dokument",
+	"Image": "Bild",
+	"Stop At": "Anhalten bei",
+	"Weight": "Gewicht",
```

Comment:
"Gewichtung" might make more sense in this context.

Reply:
This suggestion has now been implemented